settings.gradle.kts

```diff
@@ -1,11 +1,19 @@
-rootProject.name = "kover-gradle-plugin"
+rootProject.name = "kover"
 
 pluginManagement {
-    val kotlinVersion: String by settings
+    includeBuild("build-logic")
+
     plugins {
-        kotlin("jvm") version kotlinVersion
+        kotlin("jvm") version embeddedKotlinVersion
```

Comment:
Is `embeddedKotlinVersion` a variable provided by Gradle itself? I didn't notice its definition

Reply:
Yes, https://github.com/gradle/kotlin-dsl-samples/issues/94